src/main/java/duke/task/Item.java

```diff
@@ -0,0 +1,155 @@
+package duke.task;
+
+
+import java.util.Date;
+
+/**
+ * This is the main superclass that contains all the attributes of the tasks.
+ */
+public class Item {
+
+    /**
+     * status: whether Item is complete or not.
+     */
+    private Boolean status;
```

Comment:
Your variable name isn't descriptive nor indicative of what it is meant for. Refer to the coding style for a more appropriate name.